.changeset/afraid-rocks-design.md

```diff
@@ -2,11 +2,7 @@
 "@replayio/test-utils": patch
 ---
 
----
-
-## "@replayio/test-utils": patch
-
 pr: #647
 commit: f1e815a
 
-Added info and error log calls to the test-utils package.
```

Comment:
this message gets converted to a bullet point in the [test-utils' CHANGELOG](https://github.com/replayio/replay-cli/blob/40a1ec8a828b398605c3855746d675bea3090d0c/packages/test-utils/CHANGELOG.md) so mentioning the name of the package here is redundant.